src/mp-instance.js

```diff
@@ -1361,15 +1362,17 @@ function completeSDKInitialization(apiKey, config, mpInstance) {
         );
     }
 
-    mpInstance._Store.isInitialized = true;
+    // If for some reason the identity call inside of Session Manager does not run
+    // we should clear out the ready queue.
```

Comment:
Your PR description appropriately states when this is the case, rather than "for some reason", so I'd turn that into a comment.

"Will also continue to clear out the ready queue as part of the initial init flow if an identify request is unnecessary, such as if there is an existing session".